### DISCUSSION TITLE
Add styling to kbd shortcode keyboard shortcuts in Revealjs presentations

### DISCUSSION THREAD

**Discussion by c-zippel** [quarto-dev/quarto-cli#6009] [Feature Requests]
### Description

It'd be neat if keyboard shortcuts produced with the `kbd` shortcode in Reveal presentations were styled similarly to those in HTML documents, with a grey background and outline, rather than as normal slide text. 

In HTML: 
![image](https://github.com/quarto-dev/quarto-cli/assets/41382462/0c1d1168-91f3-41c3-8545-5c2628207ddf)

In Reveal:
![image](https://github.com/quarto-dev/quarto-cli/assets/41382462/6044da0e-ce62-4230-8b80-8f87ab72cddf)

Thank you for considering this feature request!

<details>
<summary>.qmd for the screenshots</summary>

````
---
format: 
  html: default
  revealjs: 
    output-file: revealjs.html
---

## Keyboard Shortcut

Here's a keyboard shortcut: {{< kbd Ctrl+S >}}

````
</details>

**Comment by cderv** [comment]
Thanks for the report. Definitely something that should be available for reveal too. 

For now the styling is tied to boostrap output 
https://github.com/quarto-dev/quarto-cli/blob/99a56e4a1e60876293794d53aa6b084a79b58622/src/resources/formats/html/bootstrap/_bootstrap-rules.scss#L195-L203

I'll look on how to port that correctly to be available for reveal js too. 

Note to self: The `kbd` shortcode extension has a CSS but it is empty
https://github.com/quarto-dev/quarto-cli/blob/41501dd86dee2103cb5d88954303feed32f1d3b0/src/resources/extensions/quarto/kbd/resources/kbd.css#L1-L2

**Reply by mcanouil** [reply]
To note, some related opened issues:

- #5879
- #5403

**Comment by mcanouil** [comment]
Closing this in favour of the opened issue:

- #6012
